app/src/main/java/com/example/superheroapi/data/AppContainer.kt

```diff
@@ -51,9 +52,6 @@ class DefaultAppContainer : AppContainer {
         retrofit.create(SuperHeroApiService::class.java)
     }
 
-    /**
-     * DI implementation for Mars photos repository
-     */
     override val superHeroRepo: SuperHeroRepository by lazy {
```

Comment:
Removed some of your copy-paste LOL! Copy pasting is a part of the job, yes, but don't forget to review what you've copy-pasted and make sure it makes sense.